README.md

```diff
@@ -1,55 +1,20 @@
-# onecx-tenant-svc
+# OneCX-tenant-svc
 
-This project uses Quarkus, the Supersonic Subatomic Java Framework.
-
-If you want to learn more about Quarkus, please visit its website: https://quarkus.io/ .
+In the OneCX Portal we need to support Multi-Tenancy. In order to do so this application was created:
```

Comment:
It sounds like someone forced you. You should more say something like: "This is the service of OneCX which is providing support for...."

Reply:
Thats a paragraph copied from the epic.

Changed to "want"